packages/client/lib/sync/txpool.ts

```diff
@@ -419,6 +427,92 @@ export class TxPool {
     })
   }
 
+  /**
+   * Helper to return a normalized gas price across different transaction types.
+   * @param tx The tx
+   * @param priority If a 1559 tx and true, returns the maxPriorityFeePerGas. If false, returns the maxFeePerGas. (default: true)
+   */
+  private txGasPrice(tx: TypedTransaction, priority = true) {
+    if (tx.supports(Capability.EIP1559FeeMarket)) {
+      if (priority) {
+        return (tx as FeeMarketEIP1559Transaction).maxPriorityFeePerGas
```

Comment:
Just re-asking here since I can't really imagine a situation where it would make sense to compare the priority fee with the classic gas price of a non-1559 tx: this is correct that it is not "max fee + max priority fee"?

Reply:
yes thanks you made me realize I need the `baseFee` here to properly return the normalized priority tip. Omitting it will return the max total fee, used for excluding txs below that amount for assembling the current block. You can see it's used for exactly those two uses in `txsByPriceAndNonce()`. I'll try to improve the clarity on the typedoc too, let me know if it's more understandable.